include/odp/api/spec/classification.h

```diff
@@ -135,6 +135,13 @@ typedef struct odp_cls_capability_t {
 
 	/** A Boolean to denote support of PMR range */
 	odp_bool_t pmr_range_supported;
+
+	/** Support for Random Early Discard */
+	odp_support_t random_early_discard;
```

Comment:
The correct term is Random Early Detection, not Discard. See [RFC 7567](https://tools.ietf.org/html/rfc7567).